.travis.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     - stage: test
       name: "Unit Testing"
       script:
-        - make test-unit-cpp-debian
+        - travis_wait 60 make test-unit-cpp-debian
```

Comment:
kk. And on second thought this is only necessary for Propagated because its unit tests take more than 10 minutes, this PR without propagated doesn't actually need the travis wait command